### PR TITLE
doc: polish the documentations on ALS and MS

### PR DESCRIPTION
**Pull request by kezhenxu94**
Polish the doc on ALS and MS as they are outdated

### CODE REVIEW

File: docs/en/setup/envoy/metrics_service_setting.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+Envoy defines a gRPC service to emit the metrics, whatever implements this protocol can be used to receive the metrics.
+SkyWalking has a built-in receiver that implements this protocol so that you can configure Envoy to emit its metrics to SkyWalking.
+
+As an APM system, SkyWalking does not only receive and store the metrics emitted by Envoy, it also analyzes the topology of services and service instances.
+
+**Attention:** There are two versions of Envoy metrics service protocol up to date,
+[v2](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/grpc_service.proto#envoy-api-msg-core-grpcservice) and
+[v3](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/metrics_service.proto), SkyWalking (8.3.0+) supports both of them.
+
+## Configure Envoy to send metrics to SkyWalking without Istio
```
Comment (wu-sheng) [inline docs/en/setup/envoy/metrics_service_setting.md:12]: Will the name format mechanism fail if there is no Istio?

File: docs/en/setup/envoy/als_setting.md [hunk 2]
```diff
@@ -48,9 +60,31 @@ helm repo add elastic https://helm.elastic.co
...
+
+`k8s-mesh` uses the metadata from Kubernetes cluster, hence in this analyzer OAP needs access roles to `Pod`, `Service`, and `Endpoints`.
+
+The [blog](https://skywalking.apache.org/blog/2020-12-03-obs-service-mesh-with-sw-and-als/) illustrates the detail of how it works, and a step-by-step tutorial to apply it into the `bookinfo` application.
+
+### `mx-mesh`
+
+`mx-mesh` uses the Envoy metadata exchange mechanism to get the service name, etc.,
+this analyzer requires Istio to enable the metadata exchange plugin (you can enable it by `--set values.telemetry.v2.enabled=true`,
+or if you're using Istio 1.7+ and installing it with profile `demo`/`preview`, it should be enabled then).
```
Comment (wu-sheng) [inline docs/en/setup/envoy/als_setting.md:89]: Consider to add both of your writes at the end of this doc again, referencing to SkyWalking website's version.

File: docs/en/setup/envoy/als_setting.md [hunk 3]
```diff
@@ -48,9 +60,31 @@ helm repo add elastic https://helm.elastic.co
...
+helm install 8.1.0 skywalking -n istio-system \
+  --set oap.env.SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=k8s-mesh \
+  --set fullnameOverride=skywalking \
+  --set oap.envoy.als.enabled=true
 ```
 
-Notice, only use this when Envoy is under Istio's control, and they are in k8s environment. The OAP requires the read right to k8s API server for all pods IPs.
+You can use `kubectl -n istio-system logs -l app=skywalking | grep "K8sALSServiceMeshHTTPAnalysis"` to ensure OAP ALS `k8s-mesh` analyzer has been activated.
+
+A more detailed tutorial can be found [on the website](https://skywalking.apache.org/blog/2020-12-03-obs-service-mesh-with-sw-and-als/).
```
Comment (wu-sheng) [inline docs/en/setup/envoy/als_setting.md:71]: This duplicates with L83, consider to keep one only?

File: docs/en/setup/envoy/metrics_service_setting.md (hunk 1 above)
Comment (kezhenxu94) [reply docs/en/setup/envoy/metrics_service_setting.md:12]: The name format still works w/o Istio if the metadata is injected, check line 55

**Review by wu-sheng — DISMISSED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**